src/main/java/seedu/duke/command/ListCommand.java

```diff
@@ -30,10 +30,10 @@ public void execute() throws Exception {
             list += "\n" + ++patientCount + ". " + patientID;
         }
 
-        // Todo implement more exceptions
         if (patientCount == 0) {
             throw new Exception(Constants.EXCEPTION_LIST_EMPTY);
         } else {
+            assert list != null : "List should not be empty";
```

Comment:
You could change "not be empty" to "not be null" to reduce confusion between list being `null` or an empty string "".

Reply:
I have done the change to ""